src/bitwarden/auth.rs

```diff
@@ -1,18 +1,32 @@
 use color_eyre::eyre::eyre;
 use serde::Deserialize;
 use serde_json::json;
+use serde_repr::{Deserialize_repr, Serialize_repr};
 
-use super::config::BwConfig;
+use super::config::ConfigResponseModel;
+
+#[derive(Serialize_repr, Deserialize_repr, PartialEq, Clone, Debug)]
+#[repr(u8)]
+pub enum KdfType {
+    Pbkdf2Sha256 = 0,
+    Argon2id = 1,
+}
+
+impl Default for KdfType {
+    fn default() -> Self {
+        KdfType::Pbkdf2Sha256
+    }
+}
```

Comment:
is this needed? i see that it's used by `derive(Default)` in `PreloginResponseModel` but im not sure if that's even needed

Reply:
it isn't. fixed